stock_auto_move/models/stock_location_path.py

```diff
@@ -1,15 +1,23 @@
 # -*- coding: utf-8 -*-
 # © 2014-2015 NDP Systèmes (<http://www.ndp-systemes.fr>)
 
-from openerp import api, models
+from openerp import api, fields, models
 
 
 class StockLocationPath(models.Model):
 
     _inherit = 'stock.location.path'
 
+    auto_confirm = fields.Boolean(
+        help="If this option is selected, the generated moves will be "
+             "automatically processed as soon as the products are available. "
+             "This can be useful for situations with chained moves where we "
+             "do not want an operator action."
+    )
```

Comment:
Although I do understand the main subject of the PR and agree with it, I do not understand why you modify the `stock.location.path` code logic. It seems to me unrelated to the subject of this PR. Maybe it could be in another PR ?

Reply:
Hi @npiganeau thanks for the review, but this PR is based on #359  and that's the reason why you see those commits here. I had to do so because the migration of the module wasn't merged yet

Reply:
Ok, I understand.